book/website/community-handbook/coworking/coworking-organisation.md

```diff
@@ -0,0 +1,131 @@
+Additionally, you should also communicate the basic etiquettes, such as muting microphones when not speaking or not interrupt others when working in a Pomodoro session unless necessary.
+
+It is useful to let everyone know who they can contact if they need help in troubleshooting during the call or need more time to discuss their ideas.
+Create an agenda that states clearly what is expected of them, such as if they should bring their tasks and questions in the call or communicate them beforehand.
+In _The Turing Way_, we have dedicated slots for both group discussions and quiet working.
+
+
+## Some Examples
+
+### Open calls for building connections
+
+One of _The Turing Way_ contributors, Natalie Thurlby has adapted the Collaboration Café to create a [Data science online co-working group](http://bristol.ac.uk/golding/ask-jgi/data-science-online-co-working-group/).
+They have designed their call format for the Jean Golding Institute community members in Bristol to work together and build connections while working on their data science projects.
+This is a great example of using coworking calls for community building and supporting others with similar research interests.
+
+### Small groups for supporting each other
+
+You can also plan your coworking calls with a small group of (2-3) colleagues with whom you can mutually hold accountability for your time and work.
+A small group reduces the burden of coordinating multiple time zones and finding a schedule that is suitable for everyone.
+Naomi Penfold and Stefanie Butland have written about their 1:1 coworking format in this post: [Online Co-working Partnerships are Community of Practice in Action](https://www.cscce.org/2020/02/04/online-co-working-partnerships-are-community-of-practice-in-action/#more-3009).
+While reviewing this chapter, Naomi described their 1:1 call format in contrast with _The Turing Way_ group format as follows:
+
+> In the co-working calls I have, it's 1:1 and we often spend 20 minutes in the beginning catching up and talking about our progress since last time.
+If there is something we want to discuss more intensely, we allocate 30 minutes of our call time to that, for example, one of us asking the other how to troubleshoot something.
+For the time in between, we do a few quiet working sessions (sometimes 25 minutes, sometimes longer).
+This is likely a combination of a co-mentoring call and a co-working Pomodoro call, and it works well for me.
+
+You can read an adapted version of their blog post along with other ideas for virtual events in the {ref}`guidebook by Center for Scientific Collaboration and Community Engagement <CSCCE2020virtual>`
+
+If you organise or host a coworking call in your community, you are welcome to highlight them here by creating a [pull request](https://github.com/alan-turing-institute/the-turing-way/pulls) on the GitHub repository.
```

Comment:
maybe clarify that here = this chapter and also link to the contributing guidelines?